test/parser/controlFlow/__snapshots__/If.test.js.snap

```diff
@@ -64,7 +64,7 @@ Array [
         },
         "file": "",
         "start": Object {
-          "column": 21,
+          "column": 20,
```

Comment:
What are these snapshots?

Reply:
I've been using [jest snapshots](https://jestjs.io/docs/en/snapshot-testing#snapshot-testing-with-jest) for the parser tests (probably since the first few parser tests that existed) to serialize an AST from the parser and compare it to a previously-serialized copy.  This fails a test when the AST changes the same way explicit `expect()`s would, but allows us to quickly update the expected state with `yarn test -u`.  It's still a [change detector test](https://testing.googleblog.com/2015/01/testing-on-toilet-change-detector-tests.html) but it feels like an appropriate use for that 😄